movie-app/src/components/switchers-buttons/SwitchersButtons.tsx

```diff
@@ -0,0 +1,35 @@
+import React, { useState } from "react";
+import SwitchersButtonWrapper from "./SwitchersButtonWrapper";
+
+const SwitchersButtons = (props: any) => {
+  const [buttons, updateSearchButtons] = useState(props.searchButtonsParams);
+
+  const activateButton = (index: number) => {
+    const newArr = [...buttons];
+    newArr.forEach((item, i) =>
+      i === index ? (item.active = true) : (item.active = false)
+    );
+    updateSearchButtons(newArr);
+  };
+
+  return (
+    <div className="search-buttons-container">
+      <span>{props.switchersTitle}</span>
+      {buttons.map((button: any, index: number) => {
+        return (
+          <SwitchersButtonWrapper
+            onClick={() => activateButton(index)}
+            key={index}
```

Comment:
you can use `button.value` as a key instead

Reply:
Fixed